.github/workflows/continuous-integration-conda.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           pip install --no-deps git+https://github.com/LSSTDESC/descwl-shear-sims.git
           pip install --no-deps git+https://github.com/LSSTDESC/descwl_coadd.git
           pip install --no-deps git+https://github.com/esheldon/metadetect.git
+          pip install --no-deps git+https://github.com/esheldon/meds.git
```

Comment:
meds is on conda - push a release if you need the master version

https://github.com/conda-forge/meds-feedstock